Set the correct region, increased timeout and use smaller num within instruction

nickksun opened the pull request:
Created a "serverless.base.yml" file as an immutable template, in case the "serverless.yml" has been wrongly changed then re-used as base file.

Set the value for "region" in "serverless.yml" to make sure the resources being created in the specified region.

From the instruction, reduced value from 100 to 10 to prevent "timeout" issue when running "executor" for "128MB" alias in an execution of the state machine.

Increased timeout from 30 seconds to 5 minutes for all functions. (Maybe only necessary for "executor"?)

alexcasalboni commented:
That's awesome, thanks for contributing! 🎉 

> Created a "serverless.base.yml" file as an immutable template, in case the "serverless.yml" has been wrongly changed then re-used as base file.

This sounds like a good idea. Should we mention that you should always work on `serverless.base.yml` instead of `serverless.yml` in the doc? That's not a framework convention, so we'd better mention it.

Also, should we remove `serverless.yml` from the repo? (this will also make sure nothing can be deployed without running the generate command)

> Set the value for "region" in "serverless.yml" to make sure the resources being created in the specified region.

That makes sense. Should we also print out a warning in case the given value differs from the value specified in `serverless.base.yml`? (something as simple as `console.log("Overriding region from A to B");`)

> From the instruction, reduced value from 100 to 10 to prevent "timeout" issue when running "executor" for "128MB" alias in an execution of the state machine.

Yes, this makes sense since `enableParallel` is `false` by default.

> Increased timeout from 30 seconds to 5 minutes for all functions. (Maybe only necessary for "executor"?)

Yes, this should be necessary only for the `executor` Function :)

nickksun commented:
Thanks for the quick reply, I've added some new commits:

```serverless.yml``` is now part of ```.gitignore``` and removed from this repo.

"Target Region: " will show after ```serverless.yml``` is generated.

Default region has been aligned to ```us-east-1``` from both base yml file and generator.

Default timeout for functions other than ```executor``` has been set to 60 seconds.

alexcasalboni commented:
LGTM :) 🎉
